### PR TITLE
added kotlin file extension

### PR DESCRIPTION
**Pull request by alkeryn**
Added the file extension for the kotlin programming language

### CODE REVIEW

**Comment by janlazo** [comment]
[vim-patch:8.2.0190](https://github.com/vim/vim/commit/ab067a21b9622513ed75f4801b001606eeaf2474) does the same but includes a test. Run `./scripts/vim-patch.sh` to create a new commit for the vim-patch.

**Comment by Alkeryn** [comment]
@janlazo i'm the author of that modification ^

File: runtime/filetype.vim.rej [hunk 1]
```diff
@@ -0,0 +1,15 @@
+--- runtime/filetype.vim
```
Comment (janlazo) [inline runtime/filetype.vim.rej]: `*.rej` files should not be committed.

```
-" Dockerfilb; Podman uses the same syntax with name Containerfile
+" Dockerfile; Podman uses the same syntax with name Containerfile
```
suggests that an earlier runtime patch is required but it's fine. These typo changes happen a lot.

Comment (Alkeryn) [reply runtime/filetype.vim.rej]: Oh ok gonna fix it wait a sec ^

**Pull request merged**